src/demo/view-configs/hubmap.js

```diff
@@ -0,0 +1,32 @@
+import { vapi } from '../utils';
+
+function getConfig() {
```

Comment:
This example never loads the full genomic profiles for me (FF browser):
<img width="1464" alt="Screen Shot 2021-03-25 at 4 30 27 PM" src="https://user-images.githubusercontent.com/43999641/112539491-718b7b80-8d87-11eb-894e-7fd3ea34c035.png">

Reply:
I think the higlass server is currently down http://higlass.io/app and that gene annotations track loads from that server